cli/testdata/coder_server_--help.golden

```diff
@@ -16,6 +16,12 @@ Start a Coder server
           $CACHE_DIRECTORY is set, it will be used for compatibility with
           systemd.
 
+      --disable-owner-workspace-exec bool, $CODER_DISABLE_OWNER_WORKSPACE_EXEC
```

Comment:
Maybe `--disable-owner-workspace-access` would be a bit clearer? I'm thinking maybe bc it's not just disabling shell access but apps as well.